solr/bin/solr

```diff
@@ -2172,6 +2172,12 @@ function start_solr() {
     SOLR_OPTS+=($AUTHC_OPTS)
   fi
 
+  # If a heap dump directory is specified, enable it in SOLR_OPTS
+  if [[ -n "$SOLR_HEAP_DUMP_DIR" ]]; then
```

Comment:
I kind of like the idea of having a boolean flag to turn on heap dumps, and have a default location in the logs directory. This logic would be fine, but having an additional check above that checks that `SOLR_HEAP_DUMP_DIR` is empty and `SOLR_HEAP_DUMP` is true, then sets `SOLR_HEAP_DUMP_DIR="${SOLR_LOG_DIR}/dumps"`.

Just a thought, for ease of use.